SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -415,7 +415,85 @@ Type 4 conversion equations are intended for data produced by Simrad EK80 and ES
 
 == Coordinate systems
 
-=== Platform coordinate system
+This section provides a complete mathematical overview of the coordinate systems necessary to physically locate the position of an echo relative to a fixed global coordinate system when measured from a moving or stationary platform. The coordinate systems detailed here follow those used in some multibeam bathymetric mapping sonars, but in many cases this level of preciseness will be unnecessary and several coordinate systems will overlap.
+
+There are five right-handed Cartesian coordinate systems (<<coordinateSystemList>>) and one Cartesian angular coordinate system used to specify the physical location of received echoes. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
+
+
+.List of coordinate systems used to physically locate the position of received echoes.
+[[coordinateSystemList]]
+
+|===
+|Coordinate system name|Origin|_x_-axis|_y_-axis|_z_-axis
+
+|Fixed coordinate system|Fixed somewhere on the nominal sea surface|Pointing North|Pointing East|Pointing down with gravity
```

Comment:
To be honest readers always had troubles with this coordinate system FCS. It is a view of mind but is never really used (it is in kmall documentation) because when used this fixed point is moving constantly (position of the ship at ping time). It has two aims : 

- switch from a vessel x,y axis coordinate reference system to a geographic one (ie latitude, longitude)

- switch depth from a surface reference system to an absolute one (ellipsoid, LAT or MSL). 

But these changes in coordinates cannot be done only with matrix and translation. And it is hard to define this csr as a center with three axis

I would suggest either to rename FCS to Geographic Coordinate System (or Absolute Coordinate System) and define it as :
 |Geographic Coordinate System|Center of the earth and sea water surface model| Longitude coordinate|Latitude coordinate|Pointing down with gravity 

or to define it as another kind of coordinate system, I mean remove it from the table and add a separate description to introduce the use of water_level and vertical_offset variables for depth computation